.github/workflows/update_packages.yml

```
@@ -48,7 +49,10 @@ jobs:
               git restore .
               Remove-Item built_pkgs -Recurse -ErrorAction Ignore
             }
-            if ($newVersion) {
+            if ($newVersion -and ($finalUpdateType -eq 'DYNAMIC_URL')){
+              git commit -m "Fix broken hash in $package"
+            }
+            elseif ($newVersion) {
```

[nitpick] I think it reads nicer in two `ifs`:
```suggestion
            if ($newVersion) {
              if ($finalUpdateType -eq 'DYNAMIC_URL') {
                git commit -m "Fix broken hash in $package"
              }
              else {
```